llm/server.go

```diff
@@ -746,7 +746,7 @@ func (s *llmServer) Completion(ctx context.Context, req CompletionRequest, fn fu
 
 	// TODO (parthsareen): Move conversion to grammar with sampling logic
 	// API should do error handling for invalid formats
-	if req.Format != nil {
```

Comment:
if reg.Format != "" { // Do something if Format is not empty fmt.Println(":", reg.Format) } else { // Handling if Format is empty fmt.Println(."") }